pulser-simulation/pulser_simulation/simulation.py

```diff
@@ -202,7 +209,7 @@ def set_config(self, cfg: SimConfig) -> None:
                 )
             # Probability of phase (Z) flip:
             # First order in prob
-            prob = self.config.dephasing_prob / 2
+            prob = self.config.dephasing_prob
```

Comment:
Why this change? Isn't this still a first-order approximation? Nothing else seems to have changed.

Reply:
There is no reason to divide by two, the default probability is already less than 0.1, so I removed it.

Reply:
Are you sure that's the reason why it was being divided by two? It seems to me that it was rather because we are using a first order approximation (which assumes p is small) and the first term in the expansion has `p/2`. This is what the code below seems to imply.

Reply:
This was added in PR #233. 
`p/2` comes from the dephasing formula, actually - see eq. (2.3) in [this paper](https://arxiv.org/pdf/1509.02921.pdf).